docs/builtin_elements.md

```diff
@@ -463,7 +463,8 @@ The FocusScope exposes callback to intercept the pressed key when it has focus.
 
 ```60
 Example := Window {
-    FocusScope {
+    forward-focus: MyKeyHandler;
+    MyKeyHandler := FocusScope {
```

Comment:
```suggestion
    forward-focus: my_key_handler;
    my_key_handler := FocusScope {
```

Our current coding style kind of use lower case for id.  (but we didn't really define that anywhere though)

Reply:
Done